source/jormungandr/tests/stif_tests.py

```diff
@@ -139,8 +140,18 @@ def test_stif_max_successive_buses_with_tram_in_between(self):
         eq_(len(response['journeys']), 1)
 
         #As we modifiy the value of _max_successive_buses to 5 we want two journeys
-        query = "journeys?from={from_sp}&to={to_sp}&datetime={datetime}&_override_scenario=stif&_max_successive_buses=5"\
+        query = "journeys?from={from_sp}&to={to_sp}&datetime={datetime}&_override_scenario=stif" \
+                "&_max_successive_buses=5&_max_additional_changes=10"\
             .format(from_sp="stopP", to_sp="stopW", datetime="20140614T145500")
 
         response = self.query_region(query, display=False)
         eq_(len(response['journeys']), 2)
+
+        # As we modifiy the value of _max_additional_changes to 2 we delete the second journey because
+        # it contains more then nb_correspondence + 2 ()
+        query = "journeys?from={from_sp}&to={to_sp}&datetime={datetime}&_override_scenario=stif&_max_successive_buses=5"\
```

Comment:
since you use the _max_additional_changes=2, I think it's better to set it in the request, not to depend on default values